src/scanpy/tools/_ingest.py

```diff
@@ -119,16 +116,6 @@ def ingest(
     >>> sc.tl.ingest(adata, adata_ref, obs="cell_type")
 
     """
-    # anndata version check
-    anndata_version = pkg_version("anndata")
-    if anndata_version < ANNDATA_MIN_VERSION:
-        msg = (
-            f"ingest only works correctly with anndata>={ANNDATA_MIN_VERSION} "
-            f"(you have {anndata_version}) as prior to {ANNDATA_MIN_VERSION}, "
-            "`AnnData.concatenate` did not concatenate `.obsm`."
-        )
-        raise ValueError(msg)
```

Comment:
this is unrelated, but since I removed another outdated version check I thought I might as well.